travel-event-host-app/src/components/header/Header.tsx

```diff
@@ -0,0 +1,146 @@
+'use client';
+import { useState } from 'react';
+import styles from './styles.module.css';
+import Link from 'next/link';
+import Avatar from '@mui/material/Avatar';
+import CircularProgress, { circularProgressClasses } from '@mui/material/CircularProgress';
+
+export default function Header() {
+  const [lang, setLang] = useState('EN');
+  const [status, setStatus] = useState('unauthenticated');
```

Comment:
Same here. create an `Enum` for the authentication status. 
What do you think about being changing `status` to `authenticationStatus` 
```suggestion
  const [status, setStatus] = useState<AuthenticationStatus>(AuthenticationStatus.Unauthenticated);
```

Reply:
that is a mockup data I will use the function useSession of NextAuth to get the user status, status: enum mapping to three possible session states: "loading" | "authenticated" | "unauthenticated"